src/nvim/change.c

```diff
@@ -343,8 +343,7 @@ static void changed_common(buf_T *buf, linenr_T lnum, colnr_T col, linenr_T lnum
               || (wp->w_topline >= lnum
                   && wp->w_topline < lnume
                   && win_linetabsize(wp, wp->w_topline, ml_get(wp->w_topline), MAXCOL)
```

Comment:
```suggestion
                  && win_linetabsize(wp, wp->w_topline, ml_get_buf(wp->w_buffer, wp->w_topline), MAXCOL)
```

Reply:
Lol, should have done this tomorrow not just before bed. There was another offender in move.c from the recent decurwinification.